include/dr/shp/zip_view.hpp

```diff
@@ -281,6 +296,8 @@ class zip_view : public rng::view_interface<zip_view<Rs...>> {
         rng::begin(std::get<Is>(views_))...);
   }
 
+  template <typename T> T min_many_impl_(T t) const { return t; }
```

Comment:
Looks like you could use std::min with an initializer list. I will try before merging.